src/main/kotlin/step1/domain/Operand.kt

```diff
@@ -0,0 +1,18 @@
+package step1.domain
+
+class Operand(input: String?) {
+    private val _value: Double
+
+    val value: Double
+        get() = _value
```

Comment:
`val` 불변으로 정의하면, 내부 프로퍼티로 정의할필요는 없을거같아요 :)
```suggestion
    val value: Double
```

Reply:
네넵 말씀하신 부분에 대해서 공감합니다. 사실 위처럼 private val 프로퍼티로 선언했을 때 private val로 바꾸라는 인텔리제이의 노란줄 경고(?)가 계속 나타나서 내부 프로퍼티를 썼는데 해당 프로퍼티를 조회하는 로직이 있다면 경고가 나타나지 않았었네요!!
수정하겠습니다 👍